src/components/header/header.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+
+export default class Header extends React.Component {
+    constructor () {
+        super()
+
+        this.state = {
+            text : ' '
+        };
+
+        this.pressEnter = this.pressEnter.bind(this);
+        this.onChangeValue = this.onChangeValue.bind(this);
+    }
+
+    onChangeValue(e) {
```

Comment:
could be just `onChange`, such name is clear enough and I don't think the "Value" part adds more clarification